src/pages/tokens/[tokenid].js

```diff
@@ -370,9 +371,9 @@ export default function Token(props) {
               >
                 <Icon icon={CalendarIcon} />
                 {token.created_at !== null
-                  ? `Minted on ${moment(tree.time_created).format(
-                      'MMMM Do, YYYY',
-                    )}`
+                  ? `<time dateTime={moment(tree.time_created).format('MMMM Do, YYYY',)}> ${moment(
```

Comment:
I don't think we can use template literals. I think we need the actual time element as a jsx element. Take a look at [this pr](https://github.com/Greenstand/treetracker-web-map-client/pull/1524/files). for some ideas.